TheaterSchedule.DAL/Repositories/GoogleFormUrlRepository.cs

```diff
@@ -18,6 +18,10 @@ public GoogleFormUrlRepository(TheaterDatabaseContext context)
 
         public void Add(UrlModel url)
         {
+            if(db.FormUrl.Count() != 0)

```

Comment:
for such checking better to use .Any() method

Reply:
Update implementation